kairon/shared/data/processor.py

```
@@ -340,14 +353,16 @@ def save_training_data(
             multiflow_stories: dict = None,
             bot_content: list = None,
             chat_client_config: dict = None,
+            other_collections: dict = None,
             overwrite: bool = False,
             what: set = REQUIREMENTS.copy(),
```

Ruff

Do not perform function call `REQUIREMENTS.copy` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)